core/tests/test_utils.py

```diff
@@ -2174,8 +2175,9 @@ class GenericTestBase(AppEngineTestBase):
     # can override the following class-level constant.
     AUTO_CREATE_DEFAULT_SUPERADMIN_USER: bool = True
 
-    SUPER_ADMIN_EMAIL: Final = 'tmpsuperadmin@example.com'
-    SUPER_ADMIN_USERNAME: Final = 'tmpsuperadm1n'
+    SUPER_ADMIN_EMAIL: Final = (
+            f'tmpsuperadm1n{uuid.uuid4().hex[:6]}@example.com')
+    SUPER_ADMIN_USERNAME: str = f'tmpsuperadm1n{uuid.uuid4().hex[:6]}'
```

Comment:
Updated so multiple super admins at the same time as other tests.

Reply:
Why is this needed now and wasn't needed/an issue before?

Reply:
Nevermind before it wasn't working now it is.